index.test-d.ts

```diff
@@ -1,4 +1,44 @@
 import {expectType} from 'tsd';
 import * as colors from './index.js';
 
+expectType<string>(colors.reset('x'));
+expectType<string>(colors.bold('x'));
+expectType<string>(colors.dim('x'));
+expectType<string>(colors.italic('x'));
+expectType<string>(colors.underline('x'));
+expectType<string>(colors.overline('x'));
+expectType<string>(colors.inverse('x'));
+expectType<string>(colors.hidden('x'));
+expectType<string>(colors.strikethrough('x'));
+expectType<string>(colors.black('x'));
 expectType<string>(colors.red('x'));
+expectType<string>(colors.green('x'));
+expectType<string>(colors.yellow('x'));
+expectType<string>(colors.blue('x'));
+expectType<string>(colors.magenta('x'));
+expectType<string>(colors.cyan('x'));
+expectType<string>(colors.white('x'));
+expectType<string>(colors.gray('x'));
+expectType<string>(colors.bgBlack('x'));
+expectType<string>(colors.bgRed('x'));
+expectType<string>(colors.bgGreen('x'));
+expectType<string>(colors.bgYellow('x'));
+expectType<string>(colors.bgBlue('x'));
+expectType<string>(colors.bgMagenta('x'));
+expectType<string>(colors.bgCyan('x'));
+expectType<string>(colors.bgWhite('x'));
+expectType<string>(colors.bgGray('x'));
+expectType<string>(colors.redBright('x'));
+expectType<string>(colors.greenBright('x'));
+expectType<string>(colors.yellowBright('x'));
+expectType<string>(colors.blueBright('x'));
+expectType<string>(colors.magentaBright('x'));
+expectType<string>(colors.cyanBright('x'));
+expectType<string>(colors.whiteBright('x'));
+expectType<string>(colors.bgRedBright('x'));
+expectType<string>(colors.bgGreenBright('x'));
+expectType<string>(colors.bgYellowBright('x'));
+expectType<string>(colors.bgBlueBright('x'));
+expectType<string>(colors.bgMagentaBright('x'));
+expectType<string>(colors.bgCyanBright('x'));
+expectType<string>(colors.bgWhiteBright('x'));
```

Comment:
Tests like this are in general not that useful as it's not actually testing anything. All the colors are defined the same, so if one is working, all should be.